index.css

```diff
@@ -0,0 +1,172 @@
+.todoList{
+    width: 50%;
+    height: 50%;
+    padding: 0;
+}
+
+#t1{
+    position: absolute;
+    top:0;
+    left:0;
+}
+
+#t2{
+    position: absolute;
+    bottom:0;
+    right:0;
+}
+
+
+h1{
+    font-family: "Andale Mono", monospace;
+    height: 10%;
+    width: 45%;
+    padding:0;
+    margin: 0;
+}
+
+.addTodo{
+    height: 10%;
+    width: 100%;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    margin-left: 0;
+    margin-right: 0;
+}
+
+.input{
+    width: 88%;
+    margin: 0;
+    padding-left: 2%;
+    padding-top: 0;
+    padding-bottom: 0;
+    padding-right: 0;
+    height: 100%;
+    border-radius: 5px;
+}
+
+.add{
+    width: 6%;
+    float: right;
+    margin: 0px;
+    height: 100%;
+    border-radius: 5px;
+}
+
+.options{
+    display: inline-flex;
+    margin: 10px;
+    width: 100%;
+    height: 10%;
+    justify-content: center;
+}
+
+.menu{
+    padding: 2%;
+    margin: 0;
+    width: 31%;
+    height: 100%;
+    background-color: rgb(235, 235, 235);
+}
+
+.menu.focused{
+    padding: 2%;
+    margin: 0;
+    width: 31%;
+    height: 100%;
+    background-color: rgb(199, 209, 214);
+}
+
+ul{
+    list-style-type: none;
+}
+li{
+    height: 3vh;
+    margin-top: 2vh;
+    margin-bottom: 2vh;
+    margin-left: 0;
+    margin-right: 0;
```

Comment:
Instead of using margin, you can use padding here.
This will let the hover background take the whole list element.

Reply:
and also, it can be written like this:
```
padding: 2vh 0;
```